src/tribol/interface/tribol.hpp

```diff
@@ -238,6 +303,15 @@ void registerNodalResponse( integer meshId,
                             real* ry,
                             real* rz=nullptr );
 
+/*!
+ * @brief Returns the response (RHS) vector as a ParGridFunction
+ * 
+ * @param [in] cs_id The ID of the coupling scheme with the MFEM mesh.
+ *
+ * @return mfem::ParGridFunction of the response (RHS) vector.
+ */
+mfem::ParGridFunction getMfemResponse( integer cs_id );
```

Comment:
@kennyweiss - Are you ok with these new 'getter' routines, where now tribol has to manage the memory of the mfem nodal force response grid function, the mortar gaps (see below) and the mortar pressures (see below)? This contrasts to how previously these were pointers registered with Tribol whose memory was managed by the host-code. 

@ebchin - clarify whether the mfem::ParGridFunction that is returned is defined over the entire volume par mesh or something else, just for clarity.

Reply:
@srwopschall The mesh of the returned grid function has been specified.

@kennyweiss 
- API has been updated to eliminate the possible copy (though I would like to move back to the other interface once we move to C++17).
- The mfem interface is now only built when redecomp is built, and redecomp is only built when MPI is configured.
- Response is related to both the coupling scheme and mfem mesh for the mfem interface, since a single mfem mesh is associated with a single coupling scheme (it's clearer now that `registerMfemMesh()` has been renamed to `registerMfemCouplingScheme()`)
- Each coupling scheme would be registered separately through multiple calls to `registerMfemCouplingScheme()`, so there should be no issues
- I will be developing a multiple mesh example for serac, and can back-port it as a test for Tribol in a separate PR.